homeassistant/components/dsmr_reader/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "config": {
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "mqtt_missing": "The MQTT service is required for this integration."
```

Comment:
```suggestion
```

Reply:
Removed after getting rid of the two mqtt validations.